gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/MultiLongWatermark.java

```diff
@@ -0,0 +1,150 @@
+/* (c) 2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.source.extractor.extract.kafka;
+
+import java.math.RoundingMode;
+import java.util.List;
+
+import com.beust.jcommander.internal.Lists;
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ComparisonChain;
+import com.google.common.math.LongMath;
+import com.google.common.primitives.Shorts;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+import gobblin.source.extractor.Watermark;
+
+
+/**
+ * A {@link gobblin.source.extractor.Watermark} that holds multiple long values.
+ *
+ * @author ziliu
+ */
+public class MultiLongWatermark implements Watermark {
+
+  private static final Gson GSON = new Gson();
+
+  private final List<Long> values;
+
+  /**
+   * Copy constructor.
+   */
+  public MultiLongWatermark(MultiLongWatermark other) {
+    this.values = Lists.newArrayList(other.values);
```

Comment:
Should have used `com.google.common.collect.Lists` instead of `jcommand.Lists`. Updated the import.
`Lists.newArrayList()` is used throughout Gobblin in favor of `new ArrayList()`.